test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -39,85 +39,88 @@ public CustomSearchParamTests(HttpIntegrationTestFixture fixture, ITestOutputHel
         [SkippableFact]
         public async Task GivenANewSearchParam_WhenReindexingComplete_ThenResourcesSearchedWithNewParamReturned()
         {
-            var patientName = Guid.NewGuid().ToString().ComputeHash().Substring(28).ToLower();
-            var patient = new Patient { Name = new List<HumanName> { new HumanName { Family = patientName } } };
-            var searchParam = Samples.GetJsonSample<SearchParameter>("SearchParameter");
-            searchParam.Code = "fooCode";
-
-            // POST a new patient
-            FhirResponse<Patient> expectedPatient = await Client.CreateAsync(patient);
-
-            // POST a second patient to show it is filtered and not returned when using the new search parameter
-            await Client.CreateAsync(Samples.GetJsonSample<Patient>("Patient"));
+            var randomName = Guid.NewGuid().ToString().ComputeHash().Substring(0, 14).ToLower();
+            var searchParam = Samples.GetJsonSample<SearchParameter>("SearchParameter-AppointmentStatus");
+            searchParam.Name = randomName;
+            searchParam.Url = searchParam.Url.Replace("foo", randomName);
+            searchParam.Code = randomName + "Code";
+            searchParam.Id = randomName;
```

Comment:
>searchParam.Id = randomName; [](start = 12, length = 28)

It's gonna be overriden during post, is it worth it?

Reply:
True!  This was unnecessary.